portal/components/icons/arrowRightIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import { ComponentProps } from 'react'
+
+export const ArrowRightIcon = (props: ComponentProps<'svg'>) => (
+  <svg
+    fill="none"
+    height="16"
+    width="16"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <path
+      clip-rule="evenodd"
+      d="M4.21983 11.78C4.07938 11.6394 4.00049 11.4488 4.00049 11.25C4.00049 11.0512 4.07938 10.8606 4.21983 10.72L9.43983 5.5H5.74983C5.55092 5.5 5.36015 5.42098 5.2195 5.28033C5.07885 5.13968 4.99983 4.94891 4.99983 4.75C4.99983 4.55109 5.07885 4.36032 5.2195 4.21967C5.36015 4.07902 5.55092 4 5.74983 4H11.2498C11.4487 4 11.6395 4.07902 11.7802 4.21967C11.9208 4.36032 11.9998 4.55109 11.9998 4.75V10.25C11.9998 10.4489 11.9208 10.6397 11.7802 10.7803C11.6395 10.921 11.4487 11 11.2498 11C11.0509 11 10.8602 10.921 10.7195 10.7803C10.5788 10.6397 10.4998 10.4489 10.4998 10.25V6.56L5.27983 11.78C5.1392 11.9205 4.94858 11.9993 4.74983 11.9993C4.55108 11.9993 4.36045 11.9205 4.21983 11.78Z"
+      fill="#A3A3A3"
+      fill-rule="evenodd"
+    />
+  </svg>
+)
```

Comment:
There's a component that already does this, and it is rendered when hovering links in the nav bar that point to external links. For example, hovering "Network status" shows it

<img width="219" height="77" alt="Image" src="https://github.com/user-attachments/assets/e7ed9104-921a-4e65-94f5-28a4523132b9" />

You can get it by doing

```
import { ArrowDownLeftIcon } from 'components/icons/arrowDownLeftIcon'
```

(Ok, maybe the name is not the appropiate one 😅 )

If at rendering time, it's not properly aligned, we can use CSS.. It seems to me that we can drop this file you've created and reuse that icon that already exists